celiagg/tests/test_text.py

```diff
@@ -0,0 +1,20 @@
+
+import numpy as np
+
+import celiagg as agg
+
+
+def test_font_byte_string():
+    canvas = agg.CanvasG8(np.zeros((1, 1), dtype=np.uint8))
+    gs = agg.GraphicsState()
+    transform = agg.Transform()
+
+    text_unicode = "Hello!"
```

Comment:
Any special reason why this isn't `text_unicode = u"Hello!"`? Seems like without that, Unicode text isn't getting tested on Python 2.7.